website/cue/reference/components.cue

```diff
@@ -625,7 +625,7 @@ components: {
 				let Args = _args
 
 				common:      false
-				description: "Configures the TLS options for incoming connections."
+				description: "Configures the TLS options for outgoing connections."
```

Comment:
Won't this be wrong for sources?

Reply:
It _seemed_ like sources would be using: https://github.com/vectordotdev/vector/blob/756f16ec51b892ce30159b7b7ac2309f760bdfb2/website/cue/reference/components.cue#L549

Reply:
https://deploy-preview-10394--vector-project.netlify.app/docs/reference/configuration/sources/http/#tls

vs 

https://deploy-preview-10394--vector-project.netlify.app/docs/reference/configuration/sinks/http/#tls